src/system-health/pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
+addopts = --cov=sonic_py_common --cov-report html --cov-report term --cov-report xml
```

Comment:
Looks like this was copy/pasted but not modified, as it still references `sonic_py_common`

Reply:
Fixed